gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/client/graph/GraphAutoDestroy.java

```diff
@@ -36,15 +36,15 @@ public class GraphAutoDestroy {
     @Autowired
     KubernetesClient kubernetesClient;
 
-    Logger logger = LoggerFactory.getLogger(GraphAutoDestroy.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(GraphAutoDestroy.class);
 
     @Scheduled(cron = "0 0/5 * * * ?")
     public boolean autoDestroyGraph() throws GaaSRestApiException {
 
         try {
             return deploymentHandler.onAutoGafferDestroy(kubernetesClient);
         } catch (ApiException e) {
-            logger.debug("Failed to auto destroy graph. Kubernetes client returned Status Code: {}" + e.getCode(), e);
+            LOGGER.error("Failed to auto destroy graph. Kubernetes client returned Status Code: {}",  e.getCode(), e);
```

Comment:
You need one `{}` per thing you are printing, so:
```java
LOGGER.error("Failed to auto destroy graph. Kubernetes client returned Status Code: {}, {}",  e.getCode(), e);
```

Reply:
@t92549 
I tried to print  {} -> e and it will not work. 
{} needs always specific property like e.getCode() or e.getMessage()

That is way I have {} only  for e.getCode()

Reply:
You're right exception comes after alone, sorry confused myself with all the logging stuff!